### DISCUSSION TITLE
not able to access "Link to Confluence CORTX lab section:  https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/192708735/Labs"

### DISCUSSION THREAD

**Discussion by Pradnya-786** [Seagate/cortx#1427] [General]
Not able to access "Link to Confluence CORTX lab section:[  https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/192708735/Labs](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/192708735/Labs)"
No VM available for cortex

**Comment by mukul-seagate11** [comment]
@Pradnya-786, Its actually an internal page so will not be accessible from public as its created under private space
